### ISSUE TITLE
ES7 Promise.prototype.finally() not supported in Edge 16 and 17

### ISSUE THREAD

**Issue by NotChrisPear-n** [redux-offline/redux-offline#280]
### We have an Offline First application that uses Redux Offline and axios. What we're seeing on Edge (and Edge only) is the following:

We'll fill out a React form created using formik to create a new record
When hit save, we see the PUT go out to our API and the record is created.
We don't see the UI logic we would normally expect after a successful save.
We go to create a second new record.
We fill out the formik form with the same data in the same fields and press save.
The PUT does not go through, but we are redirected to different page.
After being redirected to the new page, we see a GET for the record using its temporary ID that fails because there was never a PUT.
We refresh the page and get a "Save Created" toast message and see the PUT for the record go through.
In other words, we can successfully PUT the first time, but we don't necessarily see all of the state change we expect. We then can do the same steps, get no PUT but do see the state change.

Again, this happens ONLY in Edge. No, we cannot switch to Chrome. :)

Is anyone else seeing issues with subsequent PUTs in Edge?

After some research, it appears that there is an issue with Edge 16 and 17,  and trying to find a babel solution to replace the unsupported javascript feature (`ES7 Promise.prototype.finally()`) with a supported method.

**Comment by calumpeak** [comment]
Please see issue #269 
If your browser does not support the finally prototype, please use a shim :)
